dags/utils/gke.py

```diff
@@ -13,27 +13,31 @@ def create_gke_config(
 ):
 
     """
-    Helper function to create gke cluster definition dict.
-    See https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster
+    Helper function to create gke cluster definition dict. All fields must match
+    their protobuf definitions.
 
-    owner and team labels  can contain only lowercase letters, numeric characters,
+    See:
+        https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster
+        https://googleapis.dev/python/container/latest/gapic/v1/types.html#google.cloud.container_v1.types.Cluster
+    owner and team labels can contain only lowercase letters, numeric characters,
     underscores, and dashes. E.g. owner_label='hwoo', team_label='dataops'
 
     """
 
     cluster_def_dict = {
         "name": name,
+        "initial_node_count": None,
```

Comment:
The `GKEClusterCreateOperator` looks for this field, otherwise fails to startup. However, if it's set to an integer, then it will fail saying that it can't be set when `node_pools` are defined.